data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
+			``
+			`	"Caesar Augustus - Property of the Mitchells"`
+			`	3 Mitchell Drive`
+			`	Avalonia District, Hestia`
+			``
+			`	"That's odd," you say to Caesar. "How did you get lost on the wrong planet, huh boy?"`
+			`	You pet Caesar, who seems glad of the attention, and decide what to do.`
+			choice
+				`	(Take Caesar to his owners on Hestia.)`
+					goto takehim
+				`	(Leave Caesar here.)`
+
+			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action to leave Caesar at the park.`
+			`	As you walk off you hear Caesar whining, and as you look back you notice him slink off into the park. But you've made your decision; hopefully everything will work out for the best.`
+				decline
+
+			label takehim
+			`	"Well, come on boy. Let's go." Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`
+			`	Once back on your ship, you make sure he's fed, watered, and settled in for his interstellar journey.`
+
+	on complete
+		payment 1000
+		conversation
+			branch "transported pookie"
+				has "Pookie, Part 2: done"
+				not `outfit: "Puny"`
+			`Caesar has been very obedient the whole trip, much to your convenience. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`
+				goto ranch
+
+			label "transported pookie"
+			`Caesar has been very obedient the whole trip, unlike some other certain dogs you've transported in the past. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`
+
+			label ranch
+			`	It's another 45 minutes of driving down long winding roads, past various out buildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`
```

Comment:
Being a ranch doesn't feel right

- "massive central mansion" with footmen and butlers isn't what comes to mind when I think range.
- the demeanor of the woman doesn't feel like a ranch owner either

A 45 minute drive seems a bit long if we are already past the gate of the property.

Reply:
It is called a ranch, but it's not really one. It's probably a name the Mitchells thought was exotic; it's still a country estate at heart.

Reply:
The dog's name is "Augustus Caesar" which is a reference to the roman empire so my mind is preconditioned to envision some sort of massive stone type of architecture and that fits with the context and descriptions of this mission story. 'Mitchells Ranch' doesn't work. If it's not a ranch, why call it one?  Maybe, use Mitchells Villa or something similar. 
 But, it's just my opinion.

Reply:
I think it's really that the Mitchells have chosen to call it "Mitchell Ranch." It isn't a ranch, but they can call it whatever they want.

Reply:
Filthy rich boomers cosplaying Yellowstone.